### ISSUE TITLE
Support name_* tags in addition to alt_name_*

### ISSUE THREAD

**Issue by vincentdephily** [openstreetmap/osm2pgsql#429]
Rationale is that name_N is 170X more frequent than alt_name_N in the database, and that it is a saner convention.

http://taginfo.openstreetmap.org/search?q=name_1

**Comment by lonvia** [comment]
Do you mean in output-gazetteer? Then this is the wrong bug tracker, Nominatim feature requests should go [there](https://github.com/twain47/Nominatim/issues).

In any case, given how much [backlash](https://lists.openstreetmap.org/pipermail/talk/2014-September/070838.html) the introduction of alt_name_* received, it is unlikely that any alternative name schemas are supported unless there is clear consensus in the community and the tag is widly used. As for name_1 tags, [96% of the tags](http://taginfo.openstreetmap.org/keys/name_1#combinations) are from the tiger import, so those numbers don't really count.

**Comment by vincentdephily** [comment]
Yes, looks like I'm talking about the gazetteer, sorry I was just pointed at [that code](https://github.com/openstreetmap/osm2pgsql/blob/07db15e5e55eca63971a014ddb08bc47cd89f762/output-gazetteer.cpp#L127) during a discussion on name_N vs alt_name_N. I'd open an issue for nominatim too but seeing this one closed so quickly is a bit discouraging.

Even if you discount the tiger data, there is currently 6 times more name_1 than alt_name_1 in the db. Looking at the number of contributors, the ration is 20 to 1. Discounting tiger data isn't very appealing either, unless you're planning on a counter mechanical edit proposition to switch existing name_N to alt_name_N. If you include tiger data, there are even more name_1 tags than there are alt_name tags in the db.

I don't think the backlash you link to is that relevant :
* it started about a mechanical edit, which raised immediate red flags of its own and tainted the rest of the discussion
* it was more about "semicolons vs (alt_)name_N" than about "name_N vs alt_name_N"
* it argued about documentation, but currently wiki documentation about name_N and alt_name_N is easyer to find that documentation about semicolon
* another argument was the lack of tool support, which is the circular problem I'm trying to break here.

To sum it up: if you support alt_name_N (like you do in gazetteer), you should also support name_N. If you were to support only one of the variants (I'm not advocating that), it should be the name_N one.
